pkgs/development/libraries/pango/default.nix

```diff
@@ -84,14 +86,6 @@ stdenv.mkDerivation rec {
 
   doCheck = false; # test-font: FAIL
 
-  postInstall = lib.optionalString withDocs ''
```

Comment:
This is actually two issues, one fixed in pango 1.50.5 release and another one fixed in the Devhelp in GNOME 41, I would rather pick the [part of the patch from GNOME 42.alpha PR](https://github.com/NixOS/nixpkgs/pull/160343/commits/5c7750f776e20522c5f4a5b97664648de7d9dd14).